docs/develop/create-sso-office-add-ins-nodejs.md

```diff
@@ -28,7 +28,7 @@ This article walks you through the process of enabling single sign-on (SSO) in a
 
 * At least a few files and folders stored on OneDrive for Business in your Office 365 subscription.
 
-* A Microsoft Azure Tenant. This add-in requires Azure Active Directory (AD). Azure AD provides identity services that applications use for authentication and authorization. A trial subscription can be acquired at [Microsoft Azure](https://account.windowsazure.com/SignUp).
+* A Microsoft Azure subscription. This add-in requires Azure Active Directory (AD). Azure AD provides identity services that applications use for authentication and authorization. A trial subscription can be acquired at [Microsoft Azure](https://account.windowsazure.com/SignUp).
```

Comment:
Is this true? My understanding is that if someone just has an Office 365 subscription (from the dev program for example), they can still register apps without explicitly getting an Azure subscription

Reply:
@nokafor good point -- based on my experience while testing SSO this week, your understanding is correct. I'm able to connect to Azure using my O365 admin credentials (without ever having explicitly obtained an Azure subscription). 

FYI - the file where you've added this comment in this PR is actually [Ricky's Node.js tutorial](https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/create-sso-office-add-ins-nodejs?branch=kb-sso-quickstart) which has existed for a while now. But I did copy/paste this prereq into the SSO quick start as well (further down in this PR). I'll remove it from the quick start and create a VSO work item for @Rick-Kirkham to evaluate whether or not it's needed in this Node.js tutorial (and anywhere else it may be included in the SSO-related docs).